marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/entitySearch/exportSearchTest.sjs

```diff
@@ -21,7 +32,8 @@ function invokeOpticPlanService(schemaName, viewName, limit, structuredQuery, se
                 searchText,
                 queryOptions,
                 sortOrder: xdmp.toJSON(sortOrder).root,
-                columns: Sequence.from(columns)
+                columns: Sequence.from(columns),
+                acceptType: 'text/csv'
```

Comment:
Also can get rid of the `acceptType` param since we elected to drop it.

Reply:
removed